src/providers/fail_over.c

```diff
@@ -1438,7 +1452,10 @@ resolve_srv_done(struct tevent_req *subreq)
     }
 
 done:
-    if (ret != EOK) {
+    if (ret == ERR_SRV_DUPLICATES) {
+        tevent_req_error(req, ret);
```

Comment:
The only thing I don't understand about this patch is this - can you add a comment why we don't set the status as failed (that I understand. it actually didn't fail), but we still mark the tevent request as failed (that's confusing to me..)